packages/@aws-cdk/aws-stepfunctions-tasks/lib/glue/start-crawler-run.ts

```diff
@@ -0,0 +1,68 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Stack } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+// import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/**
+ * Properties for starting an AWS Glue Crawler as a task
+ */
+export interface GlueStartCrawlerRunProps extends sfn.TaskStateBaseProps {
+
+  /**
+   * Glue crawler name
+   */
+  readonly glueCrawlerName: string;
+
+}
+
+export class GlueStartCrawlerRun extends sfn.TaskStateBase {
+
+  protected readonly taskMetrics?: sfn.TaskMetricsConfig;
+  protected readonly taskPolicies?: iam.PolicyStatement[];
+
+
+  constructor(scope: Construct, id: string, private readonly props: GlueStartCrawlerRunProps) {
+    super(scope, id, props);
+
+    this.taskPolicies = this.getPolicies();
+
+    this.taskMetrics = {
+      metricPrefixSingular: 'GlueCrawler',
+      metricPrefixPlural: 'GlueCrawlers',
+      metricDimensions: { GlueCrawlerName: this.props.glueCrawlerName },
+    };
+  }
+
+  /**
+   * @internal
+   */
+  protected _renderTask(): any {
```

Comment:
Where is this being used?

Reply:
This class is extend the `sfn.TaskStateBase` and this base abstrac class need implement this `_renderTask()` method.

Reply:
Ah, makes sense.